lib/influxdb.js

```diff
@@ -161,378 +164,458 @@ InfluxdbBackend.prototype.logDebug = function (msg) {
+			}
+		}
+	}
 }
 
-InfluxdbBackend.prototype.enqueue = function (type, ts, key, value) {
-  var self = this;
+InfluxdbBackend.prototype.enqueue = function(type, ts, key, value) {
+	var self = this;
 
-  key = key + '.' + type + '.' + self.proxySuffix;
+	key = key + '.' + type + '.' + self.proxySuffix;
 
-  if (!self.registry[key]) {
-    self.registry[key] = [];
-  }
+	if (!self.registry[key]) {
+		self.registry[key] = [];
+	}
 
-  self.registry[key].push({value: value, time: ts});
+	self.registry[key].push({
+		value: value,
+		time: ts
+	});
 }
 
-InfluxdbBackend.prototype.flushQueue = function () {
-  var self = this,
-      registry = self.clearRegistry(),
-      points = [];
+InfluxdbBackend.prototype.flushQueue = function() {
+	var self = this,
+		registry = self.clearRegistry(),
+		points = [];
 
-  for (var key in registry) {
-    var payload = self.assembleEvent(key, registry[key]);
+	for (var key in registry) {
+		var payload = self.assembleEvent(key, registry[key]);
 
-    self.logDebug(function () {
-      return 'Flush ' + registry[key].length + ' values for ' + key;
-    });
+		self.logDebug(function() {
+			return 'Flush ' + registry[key].length + ' values for ' + key;
+		});
 
-    points.push(payload);
-  }
+		points.push(payload);
+	}
 
-  self.httpPOST(points);
+	self.httpPOST(points);
 
-  self.logDebug('Queue flushed');
+	self.logDebug('Queue flushed');
 }
 
 
-InfluxdbBackend.prototype.clearRegistry = function () {
-  var self = this,
-      registry = self.registry;
+InfluxdbBackend.prototype.clearRegistry = function() {
+	var self = this,
+		registry = self.registry;
 
-  self.registry = {};
+	self.registry = {};
 
-  return registry;
+	return registry;
 }
 
-InfluxdbBackend.prototype.assembleEvent_v08 = function (name, events) {
-  var self = this;
+InfluxdbBackend.prototype.assembleEvent_v08 = function(name, events) {
+	var self = this;
 
-  var payload = {
-    name: name,
-    columns: Object.keys(events[0]),
-    points: []
-  };
+	var payload = {
+		name: name,
+		columns: Object.keys(events[0]),
+		points: []
+	};
 
-  for (var idx in events) {
-    var event = events[idx],
-        points = [];
+	for (var idx in events) {
+		var event = events[idx],
+			points = [];
 
-    for (var cidx in payload.columns) {
-      var column = payload.columns[cidx];
+		for (var cidx in payload.columns) {
+			var column = payload.columns[cidx];
 
-      points.push(event[column]);
-    }
+			points.push(event[column]);
+		}
 
-    payload.points.push(points);
-  }
+		payload.points.push(points);
+	}
 
-  return payload;
+	return payload;
 }
 
-InfluxdbBackend.prototype.assembleEvent_v09 = function (name, events) {
-  var self = this;
+InfluxdbBackend.prototype.assembleEvent_v09 = function(name, events) {
+	var self = this;
 
-  var payload = {
-    name: name,
-    timestamp: events[0]['time'],
-    fields: { value: events[0]['value'] }
-  }
+	var payload = {
+		name: name,
```

Comment:
We seems to have a regression here. During some of the last RCs to come out, "name" became "measurement" and "timestamp" became "time" in the write API. This commit changed addressed that: https://github.com/bernd/statsd-influxdb-backend/commit/926fdb75f0f46d2b11b49e97a2a385000ea68dd6. I later found in testing that v0.9.0 didn't seem to like you telling it the timestamp to use: https://github.com/bernd/statsd-influxdb-backend/commit/5bd942e2e97b835a2050d2c4c2585f7ff75c21a7. That may be a bug. At the same time, InfluxDB now handles that for you so what is gained by statsd doing it for you? I didn't see one so I removed it from the v09 event report though not the logic that v08 still used.